### DISCUSSION TITLE
HurlRunnerAt don't work but HurlRunner does.

### DISCUSSION THREAD

**Discussion by MiopeCiclope** [jellydn/hurl.nvim#211] [Q&A]
Hey!
Idk if I made a mistake in my config or if it is a bug. 
When I try to use HurlRunnerAt, I get 
hurl: not HTTP method found in the current line 4 

If I use HurlRunner to run all it works

this is my config:
`return {`
  `"jellydn/hurl.nvim",`
  `dependencies = {`
    `"MunifTanjim/nui.nvim",`
    `"nvim-lua/plenary.nvim",`
    `"nvim-treesitter/nvim-treesitter"`
  `},`
  `ft = "hurl",`
  `opts = {`
    `debug = true,`
    `show_notification = true,`
    `mode = "split",`
    `-- Default formatter`
    `formatters = {`
      `json = { 'jq' }, -- Make sure you have install jq in your system, e.g: brew install jq`
      `html = {`
        `'prettier',    -- Make sure you have install prettier in your system, e.g: npm install -g prettier`
        `'--parser',`
        `'html',`
      `},`
      `xml = {`
        `'tidy', -- Make sure you have installed tidy in your system, e.g: brew install tidy-html5`
        `'-xml',`
        `'-i',`
        `'-q',`
      `},`
    `},`
    `-- Default mappings for the response popup or split views`
    `mappings = {`
      `close = 'q',          -- Close the response popup or split view`
      `next_panel = '<C-n>', -- Move to the next response popup window`
      `prev_panel = '<C-p>', -- Move to the previous response popup window`
    `},`
  `},`
  `keys = {`
    `{ "<leader>m", "<cmd>HurlRunner<CR>",   desc = "Run All requests" },`
    `{ "<leader>k", "<cmd>HurlRunnerAt<CR>", desc = "Run Api request" },`
  `},`
`}`

this is my .hurl file (I thought it could be a variable issue but both requests have the same error) :

`Get all plaforms`
`GET {{base_url}}/api/platforms`
`GET http://localhost:5081/api/platforms`

**Comment by jellydn** [comment]
Do you try with latest? Sorry, I didn't notice your question?
